File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Util/servoTester.java
package org.firstinspires.ftc.teamcode.Util;

import com.bylazar.configurables.annotations.Configurable;
import com.qualcomm.robotcore.eventloop.opmode.OpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
import com.qualcomm.robotcore.hardware.Servo;

@Configurable
@TeleOp(name = "ServoTest")
public class servoTester extends OpMode {
    
    public static String c_name = "lt";
    public static String b_name = "ft";
    public static double pos = 0.0;
    public static double pos1 = 0.0;
    @Override
    public void init() {

    }

    @Override
    public void loop() {
        Servo test = hardwareMap.get(Servo.class, c_name);
        Servo test1 = hardwareMap.get(Servo.class, b_name);
        test.setPosition(pos);
        test1.setPosition(pos1);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOp/servo_tester.kt
package org.firstinspires.ftc.teamcode.TeleOp

import com.bylazar.configurables.annotations.Configurable
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import com.qualcomm.robotcore.hardware.Servo
import dev.nextftc.ftc.NextFTCOpMode
import dev.nextftc.hardware.impl.ServoEx
@Configurable
@TeleOp
class servo_tester: NextFTCOpMode() {
    init {

    }
    companion object {
        @JvmField
        var a = 0.0
    }
    private val tg: ServoEx = ServoEx("rm", -0.1)
    private val rg: ServoEx = ServoEx("lm", -0.1)


    override fun onUpdate() {
        tg.position = 0.0
        rg.position = 1.0
    }

}